src/components/addProductsForm/AddProducts.jsx

```diff
@@ -1,5 +1,181 @@
-import React from "react";
+import React, { useState } from "react";
+import { Form, Button, Row, Col, Container } from "react-bootstrap";
+import "./style.scss";
+import db from "../config/firebaseConfig";
 
 export default function AddProducts() {
-  return <div>add Products</div>;
+  const [hasSizes, setHasSizes] = useState(false);
+  const [getProductBrand, setGetProductBrand] = useState("");
+  const [getProductTitle, setGetProductTitle] = useState("");
+  const [getProductDescription, setGetProductDescription] = useState("");
+  const [getProductCategory, setGetProductCategory] = useState("");
+  const [getProductPrice, setGetProductPrice] = useState("");
+  const [getProductSizes, setGetProductSizes] = useState([]);
+
+  // Add products to the database
+  const addNewProductToDatabase = async (e) => {
+    e.preventDefault();
+    // Getting all images and push them into an array
+    const allImages = [];
+    const firstImage = e.target["firstImage"].value;
```

Comment:
actually this is one of the rare cases where i would say splitting out into a variable is superfluous. i would also actually turn this into a for loop

for (.. in "firstImage", "secondImage", "thirdImage")

finally, if you're going to hardcode strings like this, i would make them constants (magic strings) outside of the fn and reuse them that way.

line 25: you mistyped third as "thrid" :) for that reason we want to avoid typing strings like this as much as possible.